py/trtorch/ptq.py

```diff
@@ -0,0 +1,61 @@
+from typing import List, Dict, Any
+import torch
+import os
+
+import trtorch._C
+from trtorch._compile_spec import _parse_compile_spec
+from trtorch._version import __version__
+from types import FunctionType
+import tensorrt as trt
+

```

Comment:
We should be formulating this as a class `DataLoaderCalibrator` or something that at least looks like a class to the user.

Reply:
So it should feel like you are using a constructor. 

User: 
```py

calibrator = trtorch.ptq.DataLoaderCalibrator(dl, 
                                                cache_file="<PATH>", 
                                                use_cache=True, 
                                                algo_type=trtorch.ptq.EntropyCalibrator2)

```

Reply:
I think its better for it to look like the PyTorch Python API for DataLoaders which is formulated as a class. The `make_` syntax is based on how PyTorch does dataloaders in C++ and is consistent with things like shared pointers that arent in Python. I think function to function alignment with the C++ is less important than making things feel natural in their settings

Reply:
Got it.